packages/menu/src/Menu.ts

```diff
@@ -124,6 +138,67 @@ export class Menu extends SpectrumElement {
         this.removeEventListener('keydown', this.handleKeydown);
     }
 
+    public async selectOrToggleItem(item: MenuItem): Promise<void> {
```

Comment:
This workflow also needs a similar initialization as found in the `sp-action-group` element here: https://github.com/adobe/spectrum-web-components/blob/main/packages/action-group/src/ActionGroup.ts#L242

This ensures that all of the aria attributes are appropriately applied.

Reply:
Similarly, if there was a way for this to hew as closely to the functionality in `sp-action-group` it's likely that a path toward shared code here could be of benefit down the line.